### PR TITLE
Created utils file and added get_detoxify_classification function

### PR DESCRIPTION
**Pull request by SzymonOzog**

### CODE REVIEW

File: backend/oasst_backend/utils.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import requests
+
+
+def get_detoxify_classification(inputs, hf_token):
+    API_URL = "https://api-inference.huggingface.co/models/unitary/multilingual-toxic-xlm-roberta"
+    headers = {"Authorization": f"Bearer {hf_token}"}
+    payload = {"inputs": inputs}
+    response = requests.post(API_URL, headers=headers, json=payload)
```
Comment (yk) [inline backend/oasst_backend/utils.py:8]: could you add a `response.raise_for_status()`?

Comment (SzymonOzog) [reply backend/oasst_backend/utils.py:8]: updated the pr

Comment (andreaskoepf) [reply backend/oasst_backend/utils.py:8]: @SzymonOzog thanks for the function .. unfortunately there has been a slight confusion .. there is already now merged a different version of this, see https://github.com/LAION-AI/Open-Assistant/blob/main/backend/oasst_backend/utils/hugging_face.py

.. but nevermind - not much time lost on this one.

**Comment by andreaskoepf** [comment]
closing this in favor of #404

**Pull request closed without merging**